modules/influxdb/src/main/java/org/testcontainers/containers/InfluxDBContainerV1.java

```diff
@@ -0,0 +1,165 @@
+package org.testcontainers.containers;
+
+import java.util.Collections;
+import java.util.Set;
+import org.influxdb.InfluxDB;
+import org.influxdb.InfluxDBFactory;
+import org.testcontainers.containers.wait.strategy.Wait;
+import org.testcontainers.containers.wait.strategy.WaitAllStrategy;
+import org.testcontainers.utility.DockerImageName;
+
+/**
+ * See <a href="https://store.docker.com/images/influxdb">https://store.docker.com/images/influxdb</a>
+ */
+public class InfluxDBContainerV1<SELF extends InfluxDBContainerV1<SELF>> extends GenericContainer<SELF> {
+
+    public static final Integer INFLUXDB_PORT = 8086;
+
+    private static final DockerImageName DEFAULT_IMAGE_NAME = DockerImageName.parse("influxdb");
+    private static final String DEFAULT_TAG = "1.4.3";
+
+    private static final int NO_CONTENT_STATUS_CODE = 204;
+
+    @Deprecated
+    public static final String VERSION = DEFAULT_TAG;
+
+    private boolean authEnabled = true;
+    private String admin = "admin";
+    private String adminPassword = "password";
+
+    private String database;
+    private String username = "any";
+    private String password = "any";
+
+    /**
+     * @deprecated use {@link InfluxDBContainer(DockerImageName)} instead
```

Comment:
Minor copy/paste issue - I think this should be:
```suggestion
     * @deprecated use {@link InfluxDBContainerV1(DockerImageName)} instead
```